README.md

```diff
@@ -79,7 +79,8 @@ iex> DateTimeParser.parse("2019-03-11T10:30:00pm UNK")
 {:ok, ~N[2019-03-11T22:30:00]}
 
 iex> DateTimeParser.parse("2019-03-11T22:30:00.234+00:00")
-{:ok, ~U[2019-03-11T22:30:00.234Z]}
+{:ok, DateTime.from_naive!(~N[2019-03-11T22:30:00.234Z], "Etc/UTC")}
+# `~U[2019-03-11T22:30:00.234Z]` in Elixir 1.9+
```

Comment:
To run tests on older versions of Elixir (like 1.4.x) I couldn't use the `~U` sigil.